### PR TITLE
Fix UPGRADE.md table syntax so it works in GitHub

### PR DESCRIPTION
**Pull request by ntzm**
GitHub doesn't like when you miss out the starting pipe and have an empty cell first

### CODE REVIEW

**Comment by SpacePossum** [comment]
hmm odd we never spotted that before, good catch!
is there a way we can fix the layout without the `&nbsp;`, it feels odd to mix HTML with MD.

**Comment by ntzm** [comment]
You can surround the whole table with pipes:

```
| heading | foo |
| ------- | --- |
| foo     | bar |
```

**Comment by SpacePossum** [comment]
worth I try I would say :)

**Comment by SpacePossum** [comment]
looking good :)
thanks!

**Comment by keradus** [comment]
opening pipe on left side, but not closing pipe on right side ?

**Comment by ntzm** [comment]
Shall I convert the other tables as well?

**Comment by julienfalque** [comment]
@ntzm I'd say yes, for consistency.

File: UPGRADE.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ Exit codes
...
-1.x bit | 2.0 bit | Description                                                 | Note
--------:| -------:| ----------------------------------------------------------- | ----
-0       | 0       | OK                                                          |
-1       | 1       | General error (or PHP/HHVM minimal requirement not matched) | no longer used for other states, never combined with other flags
-        | 4       | Some files have invalid syntax                              | flag was added, works only in dry-run mode
-        | 8       | Some files need fixing                                      | flag was added, works only in dry-run mode
-16      | 16      | Configuration error of the application                      |
-32      | 32      | Configuration error of a Fixer                              |
-        | 64      | Exception within the application                            | flag was added
+| 1.x bit | 2.0 bit | Description                                                 | Note
```
Comment (julienfalque) [inline UPGRADE.md]: You forgot the last `|` here.

File: UPGRADE.md [hunk 2]
```diff
@@ -49,15 +49,15 @@ Exit codes
...
+| -------:| -------:| ----------------------------------------------------------- | ---------------------------------------------------------------- |
```
Comment (julienfalque) [inline UPGRADE.md]: Can you also replace the `:` here? I don't think they were intended.

**Comment by keradus** [comment]
ping @ntzm

**Comment by ntzm** [comment]
Oops, I'll fix those in an hour or so

File: UPGRADE.md [hunk 3]
```diff
@@ -49,15 +49,15 @@ Exit codes
 
 Exit codes for `fix` command have been changed and are build using the following bit flags:
 
-1.x bit | 2.0 bit | Description                                                 | Note
--------:| -------:| ----------------------------------------------------------- | ----
```
Comment (keradus) [inline UPGRADE.md]: from what I see @julienfalque asked you to remove the `:` here

they are supposed to align columns with integer to right side. I would keep that alignment.

Comment (julienfalque) [inline UPGRADE.md]: Oh I didn't know that, sorry.

Comment (keradus) [inline UPGRADE.md]: changed

**Comment by keradus** [comment]
Thank you @ntzm , really great you spot the issue and be breave to fix it yourself !

**Pull request closed without merging**